data/saas/dataset/stripe_dataset.yml

```diff
@@ -55,6 +55,8 @@ dataset:
               read_only: True
           - name: default_source
             data_categories: [system.operations]
+            fidesops_meta:
+              read_only: True
```

Comment:
I had to flag the `default_source` as read-only since it won't exist after the card is deleted and will cause an error if we post it back in the customer body.